packages/guide/best-practice.md

```diff
@@ -104,12 +106,12 @@ const title = computed(() => isDark.value ? '🌙 Good evening!' : '☀️ Good
 useTitle(title)
 ```
 
-###### Reactive Getter
+###### Reactive Getter Argument
 
-Since VueUse 9.0, we introduce a new convention for passing "Reactive Getter" as the argument. Which works great with reactive object and [Reactivity Transform](https://vuejs.org/guide/extras/reactivity-transform.html#reactivity-transform).
+Since VueUse 9.0, we introduced a new convention for passing a "Reactive Getter" as the argument, which works great with reactive objects and [Reactivity Transform](https://vuejs.org/guide/extras/reactivity-transform.html#reactivity-transform).
 
 ```ts
-const isDark = useDark()
+const config = reactive({ isDark: useDark() });
```

Comment:
Why do we need to change this?

Reply:
To illustration a more real-world example of when a getter function might be used. A ref that holds a primitive value can be passed as-is without a getter function (the getter function adds no value); however, when passing a property within a reactive object, that property's value is not reactive unless wrapped in a ref (ref, computed, etc) or returned from a getter function. @antfu 

Does this make sense? If not, it's very possible I'm misunderstanding Vue's reactivity system here 😅

Reply:
I think your understanding is correct. But giving this is just a simple example, I don't think we need to introduce an extra concept there (it might be confusing to readers why a `reactive` is needed). The getter function with conditioning already shows the use cases of an inline getter (act like a computed). For this line, I think we could keep it as-is.

Reply:
Makes sense. I've reverted this change.